python/paddle/v2/framework/tests/gradient_checker.py

```diff
@@ -109,12 +110,72 @@ def product(dim):
 
 
 class GradientChecker(unittest.TestCase):
-    def assert_is_close(self, numeric_grads, scope, max_relative_error,
-                        msg_prefix):
-        for name in numeric_grads:
-            b = numpy.array(scope.find_var(grad_var_name(name)).get_tensor())
-            a = numeric_grads[name]
+    def get_grad(self, forward_op, backward_op, input_vars, grad_names, place):
+        scope = core.Scope()
+        ctx = core.DeviceContext.create(place)
 
+        inputs = forward_op.inputs()
+        in_names = [item for k in inputs for item in inputs[k]]
+        outputs = forward_op.outputs()
+        out_names = [item for k in outputs for item in outputs[k]]
+
+        # create input var and set value
+        for name, value in input_vars.iteritems():
```

Comment:
These `input_vars` is numpy array but not paddle's Variable, is it better to name it `input_python_vars`